src/chainlit/frontend/src/components/organisms/chat/message/content.tsx

```diff
@@ -49,6 +49,10 @@ function prepareContent({ id, elements, content, language }: Props) {
     ? new RegExp(`(${elementNames.join('|')})`, 'g')
     : undefined;
 
+  if (typeof content === 'object') {
```

Comment:
Might technically be a number or boolean too. Perhaps `typeof content !== 'string'` ?

Reply:
just moved the logic to the server. it has to be a string because the message can be persisted and has to respect the DB schema